staging/src/k8s.io/code-generator/kube_codegen.sh

```diff
@@ -433,9 +443,6 @@ function kube::codegen::gen_openapi() {
 #   --plural-exceptions <string = "">
 #     An optional list of comma separated plural exception definitions in Type:PluralizedType form.
 #
-#   --prefers-protobuf
```

Comment:
why did you delete this?

Reply:
I think when i rebased i accidentally removed this comment. I will do the correction. Thanks for pointing it out.

Reply:
Fixed in 2d583db5b77e7292da7851681ebbd2e43504f2e7